### ISSUE TITLE
Vocola2 3.1.3 traceback, syntax error?

### ISSUE THREAD

**Issue by dougransom** [dictation-toolbox/Vocola2#12]
```
Package       Version
------------- -------
debugpy       1.6.3
dtactions     1.5.6
natlink       5.4.4
natlinkcore   5.3.4
pip           22.3.1
pydebugstring 1.0.0.0
pyenvutils    1.0.0.3
PySimpleGUI   4.60.4
pywin32       305
setuptools    63.2.0
unimacro      4.1.2
vocola2       3.1.3


Natlink Version: 5.4.4
Natlink pyd path: C:\Program Files (x86)\Natlink\site-packages\natlink\_natlink_core15.pyd
Use DebugView to debug natlink problems.
	https://docs.microsoft.com/en-us/sysinternals/downloads/debugview
Python Version: 3.10.3 (tags/v3.10.3:a342a49, Mar 16 2022, 12:53:06) [MSC v.1929 32 bit (Intel)]
Natlink is loaded...

Starting natlink loader from config file:
	"C:\Users\doug\.natlink\natlink.ini"
loading module: __init__
loading module: _vocola_main
created directory: {directory}
---copy AutoHotkey script "getintoforeground.ahk" from
Samples directory to "C:\Users\doug\.autohotkey"----
---copy AutoHotkey script "showmessageswindow.ahk" from
Samples directory to "C:\Users\doug\.autohotkey"----
Traceback (most recent call last):
  File "C:\Users\doug\AppData\Local\Programs\Python\Python310-32\lib\site-packages\natlinkcore\loader.py", line 305, in load_or_reload_module
    module = self._import_module_from_path(mod_path)
  File "C:\Users\doug\AppData\Local\Programs\Python\Python310-32\lib\site-packages\natlinkcore\loader.py", line 261, in _import_module_from_path
    loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\doug\AppData\Local\Programs\Python\Python310-32\lib\site-packages\vocola2\_vocola_main.py", line 50, in <module>
    from vocola2.exec.vcl2py.main import main_routine     # main.main_routine  compile function
  File "C:\Users\doug\AppData\Local\Programs\Python\Python310-32\lib\site-packages\vocola2\exec\vcl2py\main.py", line 17, in <module>
    from vocola2.exec.vcl2py.emit      import output
  File "C:\Users\doug\AppData\Local\Programs\Python\Python310-32\lib\site-packages\vocola2\exec\vcl2py\emit.py", line 41
    OUT = open(out_file, "w", encoding='latin-1')
IndentationError: unexpected indent
Traceback (most recent call last):
  File "C:\Users\doug\AppData\Local\Programs\Python\Python310-32\lib\site-packages\natlinkcore\loader.py", line 305, in load_or_reload_module
    module = self._import_module_from_path(mod_path)
  File "C:\Users\doug\AppData\Local\Programs\Python\Python310-32\lib\site-packages\natlinkcore\loader.py", line 261, in _import_module_from_path
    loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\doug\AppData\Local\Programs\Python\Python310-32\lib\site-packages\vocola2\_vocola_main.py", line 50, in <module>
    from vocola2.exec.vcl2py.main import main_routine     # main.main_routine  compile function
  File "C:\Users\doug\AppData\Local\Programs\Python\Python310-32\lib\site-packages\vocola2\exec\vcl2py\main.py", line 17, in <module>
    from vocola2.exec.vcl2py.emit      import output
  File "C:\Users\doug\AppData\Local\Programs\Python\Python310-32\lib\site-packages\vocola2\exec\vcl2py\emit.py", line 41
    OUT = open(out_file, "w", encoding='latin-1')
IndentationError: unexpected indent
loading module: __init__
loading module: _control
```

**Comment by quintijn** [comment]
too bad, I will fix it

**Comment by quintijn** [comment]
fixed, pypi release 3.1.4 now

**Comment by dougransom** [comment]
excellent.  Vocola seems to work, i did a quick test.  V3.1.5
